options/locale/locale_en-US.ini

```diff
@@ -2096,6 +2097,8 @@ settings.event_pull_request_review = Pull Request Reviewed
 settings.event_pull_request_review_desc = Pull request approved, rejected, or review comment.
 settings.event_pull_request_sync = Pull Request Synchronized
 settings.event_pull_request_sync_desc = Pull request synchronized.
+settings.event_pull_request_approvals = Pull Request Approvals
+settings.event_pull_request_merge = Pull Request Merge
```

Comment:
`Merging`?

Reply:
Or better rename section above to `Approve` so the both match grammatically.

Reply:
"merge" could be a noun. So I think it's fine?

Reply:
Right